Dockerfile

```
@@ -1,7 +1,7 @@
 FROM codait/max-base:v1.1.0
```

Can you set this to the latest while you're at it?
```suggestion
FROM codait/max-base:v1.1.1
```